Makefile.archlinux

```diff
@@ -124,6 +124,10 @@ dist-prep:
 # dist-build-dep - install build dependencies (should operate on chroot directory)
 dist-build-dep:
 	@echo "--> Archlinux dist-build-dep (makefile)"
+	@echo "  --> Generate locales..."
+	@echo "en_US.UTF-8 UTF-8" | sudo tee -a $(CHROOT_DIR)/etc/locale.gen
+	@sudo $(CHROOT_ENV) chroot "$(CHROOT_DIR)" su user -c 'sudo locale-gen'"
```

Comment:
there is no need to switch to and then back to root, just put `locale-gen` instead of `su ...`.